lib/internal/assert.js

```diff
@@ -13,10 +13,13 @@ let white = '';
 const kReadableOperator = {
   deepStrictEqual: 'Expected inputs to be strictly deep-equal:',
   strictEqual: 'Expected inputs to be strictly equal:',
+  strictEqualObject: 'Expected reference equal inputs but got:',
```

Comment:
One small suggestion here, can't we have:

`Expected "actual" reference to be equal to "expected" but got:`. For me the word `inputs` (why we use plural word here?) is bit confusing and also the counter part message (`notStrictEqualObject`) uses the term "actual" and "expected", so it would make sense to use the same here too.

Reply:
You are right that either `notStrictEqualObject` should be `'Expected inputs not to be reference equal:'` or that this message should be changed.

I used `inputs` instead of naming the variables because the message itself is significantly shorter that way. But I personally have no strong opinion on it. Maybe others can weight in? :)

@Trott @vsemozhetbyt it would be good if you could also have a look :)

Reply:
If the current wording remains, maybe this should be `reference-equal` since it’s used as a single adjective? I’ve also always been wondering about `inputs` in these error messages … it makes sense from the PoV of the `assert` module, but for me something more generic like `values` would seem more intuitive?

Reply:
No strong opinion)

Reply:
No strong opinion from me either. Maybe @nodejs/documentation even though this isn't a doc question?

Reply:
I decided to go for a more verbose version and I also added the dash in `reference-equal`.

I also added another commit to switch from `inputs` to `values`.